roles/core/display_tuning/molecule/default/converge.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Converge
+  hosts: all
+
+  vars:
+    j2_current_iceberg_number: 0
```

Comment:
We start numbering at 1, since a cluster stack is related to existing physical elements.

Reply:
done.